src/main/java/org/gelecekbilimde/scienceplatform/post/model/request/CategoryCreateRequest.java

```diff
@@ -9,10 +12,27 @@
 @Setter
 @Builder
 public class CategoryCreateRequest {
-	@NotBlank
+
+	@NotBlank(message = "Category name cannot be blank")
+	@Size(max = 36, message = "Category name must be less than 36 characters")
 	private String name;
+
+	@NotBlank(message = "Description cannot be blank")
+	@Size(max = 255, message = "Description must be less than 255 characters")
+	private String description;
```

Comment:
@aleyna-yildizli Özel bir mesaj yazmamaya çalışalımi buradaki mesajları bu şekilde özelleştirdiğimizde yazması zor olur ve uyuşmaz. Mümkün olduğunca spring'in kendi varsayılan mesajlarını kullanalım. 

Ek olarak kategori açıklaması tek karakter olabilir mi? Örneğin `C` diye bir kategori olabilir mi? Eğer olamazsa buradaki minimum değeri de belirlememizde fayda var.
```suggestion
	@NotBlank
	@Size(max = 255)
	private String description;
```